src/Uno.Templates/content/unoapp/.template.config/template.json

```diff
@@ -422,9 +422,28 @@
     },
     "http": {
```

Comment:
Maybe we wanna consider changing the `Http` long name for the cli command? It's currently use-http which conveys the idea of a bool value.

https://github.com/unoplatform/uno.templates/blob/2bce19094a415d46ca8875e602824d5c9dc7e8ee/src/Uno.Templates/content/unoapp/.template.config/dotnetcli.host.json#L64-L67